atelier-openapi/src/lib.rs

```diff
@@ -62,21 +74,18 @@ impl ModelVisitor for OpenApiModelVisitor {
         traits: &AppliedTraits,
         shape: &Service,
     ) -> Result<(), Self::Error> {
-        let mut spec = self.spec.borrow_mut();
-
         let title_trait_id = &ShapeID::from_str("smithy.api#title").unwrap();
 
-        let title = expect_string_trait_value(traits, title_trait_id)
-            .unwrap_or(format!("{}", id.shape_name()));
-
-        // let mut info = openapi3::Info::default();
-        // info.version = shape.version().clone();
-        spec.info = openapi3::Info {
-            version: shape.version().clone(),
-            title,
-            ..openapi3::Info::default()
+        let title = if traits.contains_key(title_trait_id) {
+            expect_string_trait_value(traits, title_trait_id)
+        } else {
```

Comment:
not sure if this is the best way to go about extracting traits. I'd like to [is_titled](https://github.com/johnstonskj/rust-atelier/blob/afe5ce969f4f0de2a1e46a5273bec9eb179d388f/atelier-core/src/model/shapes/mod.rs#L209) but `HasTraits` is not implemented for `Service`. Not sure how this all fits together just yet